app/src/js/components/Granules/granule.js

```diff
@@ -123,9 +123,8 @@ const metaAccessors = [
     accessor: bool,
   },
   {
-    label: 'Duplicate',
-    property: 'hasDuplicate',
-    accessor: bool,
+    label: 'Producer Granule ID',
```

Comment:
Given we call granuleId name, could this label be confusing?      This isn't a blocker as you're doing what we said we wanted to do, but this might be worth an arch/UI/PO discussion re: UI/UX.